posts/2025-04-29_why-gno/README.md

```diff
@@ -0,0 +1,272 @@
+and even the "true Christian" communities feel an urgent need for
+censorship-proof coordination and communication tools. These communities have
+influencers who are kept hidden from the general public; they have suffered
+deplatforming, defamations, and even death.
+
+Build tools, connections, and relations with these particular communities and
+especially those influencers who are nuanced in their research and speech.
+Even those that don't promote crypto will see the benefits uniquely offered by
+Gno.land.
+
+## Gno.land License
+
+Anyone can make Gno VM powered chains derived from Gno.land according to the
+viral copyleft license terms and strong attribution requirement. The Strong
+Attribution clause of the Gno Network GPL license preserves the spirit of the
+GNU AGPL license for the blockchain world.
+
+
+## Tokenomics
+
+$GNOT is the storage lock-up utility token, so Gno.land is to Gno England like
+$GNOT is to presence in Gno England, where total storage is kept finite for
+very-long-term existential purposes, and value is derived from the Gno
+artifacts created by its users, and some new users competing for attention from
+many existing users.
+
+Gno.land may migrate to AtomOne ICS once it is support hard-fork upgrades.
+There Gno.land would be one ICS shard, and many Gno VM shards may also exist,
+each with their own namespace and probably each their own storage token unless
+separate treaties are made between the main Gno.land chain (ICS shard) and
+other Gno VM shards. Transaction fees for CPU usage may be paid in either $GNOT
+or $PHOTON.
+
+## Team
+
+### New Tendermint, LLC
+
+NewTendermint, LLC is the core maintainer of the GnoVM, Tendermint2, and 
+at present Gno.land.
+
+#### GnoVM & Gno.land Core Team
+
+ * Jae Kwon before and after creating Tendermint and Cosmos always had a
+   passion for programming languages and wrote multiple parsers and
+   interpreters, and initially also wrote an EVM on top of the framework which
+   became the Cosmos SDK. Gno.land is the result of two decades of search for
+   the logoverse.
+
+ * Manfred Touron, builder focused on open-source and resilient technologies;
+   co-founded scaleway (cloud) and berty (p2p messaging), with contributions to
+   900+ open-source projects.
+
+ * Miloš Živković - Senior distributed systems engineer; passion for solving
+   protocol-level problems in the blockchain space.
+
+ * Morgan Bazalgette - Senior Go engineer; bringing the joy of developing Go to
+   Gno.
+
+ * Ray Qin - With over 15 years of experience in software development and
+   building large-scale networks, I have a deep passion for Go programming
+   language and blockchain technology.
+
+ * Marc Vertes - Senior VM and hardware developer; more than 3 decades of
+   experience, Co-founder of 3 companies (1 acquired by IBM), author of 34
+   patents, author of the Yaegi Go interpreter.
+
+ * Alexis Colin - Senior Frontend Engineer
```

Comment:
Alexis Colin - Senior Frontend Engineer with 10+ years of experience building user-focused interfaces and exploring modern tech stacks. Driven to push forward efficient UX in the blockchain space through clean code and technical precision. Currently working on gnoweb.